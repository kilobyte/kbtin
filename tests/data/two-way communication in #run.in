#hook close {#showme FIN;#end}
#run a pingpong
#act {pong %1} {#if {$1>=10} {exit} #else {#math x {$1+1};ping $x}}
ping 1
