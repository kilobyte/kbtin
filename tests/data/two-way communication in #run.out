pong 2
pong 4
pong 6
pong 8
pong 10
Not a ping!
FIN
